packages/frontend/src/components/DashboardTiles/DashboardSqlChartTile.tsx

```diff
@@ -123,6 +120,16 @@ export const DashboardSqlChartTile: FC<Props> = ({
                 tile={tile}
                 title={tile.properties.title || tile.properties.chartName || ''}
```

Comment:
```titleHref={`/projects/${projectUuid}/sql-runner/${error.slug}`}```

is it possible to add this to the error chart?

Reply:
ok last bit, 
You will need to throw the slug as well on the `getSqlChartAndResults` util

```ts

try {
        const results = await getResultsFromStream<RawResultRow>(url);
        return {
            chart: chartAndScheduledJob.chart,
            resultsAndColumns: {
                results,
                columns:
                    isApiSqlRunnerJobSuccessResponse(job) &&
                    job?.details &&
                    !isErrorDetails(job.details)
                        ? job.details.columns
                        : [],
            },
        };
    } catch {
        throw {
            ...job,
            slug,
        };
    }
```

not sure if you have something more unified but this allows us to get the slug if the results from stream fails as well!

Reply:
because at the moment (not ideal) there are 2 queries